app/views/workshops/_workshop_actions.html.haml

```diff
@@ -1,4 +1,4 @@
-%p
+%p.h4
```

Comment:
is this right? Wouldn't this mean it's a paragraph with a `h4` class attached?

Reply:
These helper classes aren't in the Foundation we're using but the latest version does. The docs are here: http://foundation.zurb.com/sites/docs/typography-helpers.html#typescale

I've added them in to our version, it's so we can have it presentationally look the same as a h4 element but the markup is still a paragraph.

Might be a better way of styling that but I think the classes make some sense, "this should look the same as a h4"?